evalml/utils/gen_utils.py

```diff
@@ -294,15 +294,11 @@ def _convert_to_woodwork_structure(data):
     # Convert numpy data structures to pandas data structures
     if isinstance(data, list):
         ww_data = np.array(data)
-
-    if isinstance(ww_data, pd.api.extensions.ExtensionArray) or (isinstance(ww_data, np.ndarray) and len(ww_data.shape) == 1):
```

Comment:
Yay, Woodwork update means we can convert directly from numpy now, so no need for this extra logic to convert to pandas first!